billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/export/saftpt/v1_04_01/PTSAFTFileGenerator.java

```diff
@@ -1160,7 +1166,8 @@ private List<PaymentMethod> getPaymentsList(PTGenericInvoiceEntity document)
                         this.MAX_LENGTH_2, true));
                 payment.setPaymentAmount(this
                         .validateBigDecimal(((PTPayment) p).getPaymentAmount()));
-                payment.setPaymentDate(this.formatDate(p.getPaymentDate()));
+                payment.setPaymentDate(this.formatLocalDate(LocalDate
+                                                           .ofInstant(document.getDate().toInstant(), document.getBusiness().getTimezone())));
```

Comment:
This is the document creation date =/= payment date

Reply:
Fixed alongside another similar error I found while reviewing all the other changes I did